docs/source/contributing/documentation.rst

```diff
@@ -235,11 +235,11 @@ Example: ``:class:`numpy.ndarray`​`` for a numpy ndarray.
 Reference type specifications
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-**The following instructions refer to types of attributes, parameters
+**The following instructions refer to types of attributes, parameters,
 and return values.** When specifying a type mid-text, it does not
 necessarily have to be typeset. However, if it's a class name, a method,
 or an enum's attribute/variant, then it is recommended to be typeset at
-least on the first occurrence of the name, so that the users can quickly
+least on the first occurrence of the name so that the users can quickly
```

Comment:
I slightly prefer the original version, but I think both are grammatically correct.

Reply:
The comma wasn't need for that part so I removed it